### DISCUSSION TITLE
Any way to implement wishlist for new features?

### DISCUSSION THREAD

**Discussion by VaZark** [directus/directus#11699] [Archive]
Hi, it's me again. ^^

The TS version of Directus is pretty young and growing fast. Is there any way we can run polls or upvote new feature requests that the community thinks is important? It'll also help the team behind directus to have a clearer idea how of it is being used since you don't collect user data and get additional data points on where to best position yourself in the market

Of course, there should not be any binding expectation that the highest polling feature is prioritised but it'll definitely help reorganise them internally; (especially since you'v been promising big changes in a few weeks)

**Comment by benhaynes** [comment]
We actually are about to start some social campaigns to ask about usage and desired features. I'm curious, are you talking about polling for existing or new features?

New feature requests are already tracked as GitHub Discussions, which have voting built in. We're always looking at the most highly rated feature requests to see what's best to work on next. Is that what you're looking for?

**Reply by VaZark** [reply]
I was under the impression that the GH Discussions are to just pitch new features, not actively track them. (since I'm unable to find a sort option based on the votes) However, that does answer my question though.

**Reply by benhaynes** [reply]
Discussions are for pitching ideas and then voting on which you think are most meaningful. They also help define the spec for smaller features (larger features should use the formal RFC process). Once they are accepted or ready to work on, they are often converted into issues.

**Reply by benhaynes** [reply]
I think sorting by "top" is their way of showing most votes... I don't really understand the logic they're using behind the scnenes though.

**Reply by rijkvanzanten** [reply]
> I don't really understand the logic they're using behind the scnenes though.

It's some odd combo between number of views, upvotes, comments, and different people who chimed in.
